src/coreneuron/apps/corenrn_parameters.hpp

```diff
@@ -58,13 +58,14 @@ struct corenrn_parameters_data {
 
     bool mpi_enable = false;         /// Enable MPI flag.
     bool skip_mpi_finalize = false;  /// Skip MPI finalization
-    bool multisend = false;          /// Use Multisend spike exchange instead of Allgather.
-    bool threading = false;          /// Enable pthread/openmp
-    bool gpu = false;                /// Enable GPU computation.
-    bool cuda_interface = false;     /// Enable CUDA interface (default is the OpenACC interface).
+    int mpi_subworld = -1;        /// MPI subworld index if ParallelContext.subworlds has been used.
```

Comment:
If this variable is not used it can be deleted.
I guess its intention is to select in which subworld CoreNEURON runs?

Reply:
Yes, this is not used and I have removed it.